community/cypher/front-end/parser/src/main/scala/org/neo4j/cypher/internal/parser/InvalidInputErrorFormatter.scala

```diff
@@ -24,9 +24,7 @@ import org.parboiled.matchers.TestNotMatcher
 import org.parboiled.support.Chars
 import org.parboiled.support.MatcherPath
 
-import scala.collection.JavaConversions.`deprecated asScalaBuffer`
-import scala.collection.JavaConversions.`deprecated seqAsJavaList`
-
+import scala.collection.JavaConverters._
```

Comment:
Our scala style does not allow wildcard imports. The build failed on our CI servers.

Reply:
Done, thanks.